src/Illuminate/Database/Connection.php

```diff
@@ -450,7 +451,7 @@ public function prepareBindings(array $bindings)
     /**
      * Execute a Closure within a transaction.
      *
-     * @param  \Closure  $callback
+     * @param  \Throwable  $callback
```

Comment:
Wait shouldn't it be `@param \Closure` instead of `@throws \Throwable`? Aren't you passing a callback function in, not an exception (which is what I understand a Throwable to be tell me if I'm wrong)